lib/util/git/index.ts

```diff
@@ -1122,6 +1129,54 @@ export async function fetchBranch(
   }
 }
 
+/**
+ *
+ * Configure git to be able to use the git-credential feature
+ *
+ * 0. Run the command: git config --global credential.helper 'store --file=localDir/.git-credentials'
+ * 1. Creates the file localDir/.git-credentials and fill it with `https://oauth2:<TOKEN>@<ENDPOINT>
+ *
+ */
+export async function configureCredentialHelperStore(
+  content: string
+): Promise<void> {
+  logger.debug('Configuring credential helper and git-credential file');
+  const gitCredentialsFile = upath.join(
+    GlobalConfig.get('localDir'),
+    '.git-credentials'
+  );
+  let res;
+  try {
+    res = await git.raw([
+      'config',
+      '--global',
```

Comment:
this will mess up global user config. I think this should be configured for local repo only.

Reply:
@viceice when I don't set `--global` to got error `fatal: not in a git directory`

so the following doesn't initialize a git repo ? 
```js
  git = simpleGit(GlobalConfig.get('localDir'), simpleGitConfig()).env({
    ...process.env,
    LANG: 'C.UTF-8',
    LC_ALL: 'C.UTF-8',
  });
```